athina/steps/loop.py

```
@@ -1,8 +1,11 @@
 import asyncio
+import json
 from typing import Dict, List, Any, Optional
```

Ruff (0.8.2)

`typing.Any` imported but unused

Remove unused import: `typing.Any`

(F401)